### ISSUE TITLE
Could not load file or assembly Octokit, Version=0.23.0.0

### ISSUE THREAD

**Issue by dario-l** [aliozgur/SeqApps#11]
Hi,
I just installed Seq.App.Github package but when I want to send event to the github the error occurs. :(

```
System.IO.FileNotFoundException: Could not load file or assembly 'Octokit, Version=0.23.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. The system cannot find the file specified.

File name: 'Octokit, Version=0.23.0.0, Culture=neutral, PublicKeyToken=null'
   at SeqApps.GitHub.GitHubIssueReactor.CreateIssue(Event`1 evt)
   at SeqApps.GitHub.GitHubIssueReactor.On(Event`1 evt)
```

**Comment by aliozgur** [comment]
@dario-l  did you install the package by following https://docs.datalust.co/v3/docs/installing-seq-apps. 
If that is the case and you still get the error the problem should be related with Seq. Please see this nuspec https://github.com/aliozgur/SeqApps/blob/master/src/Seq.Apps/SeqApps.GitHub/SeqApps.GitHub.nuspec file where the Octokit dependency is already presen

**Comment by dario-l** [comment]
Thanks for the explanation. Unfortunately I installed this app just like official docs says.

On packages folder there was Octokit but in 0.48.0 version. Not 0.23.0 like nuspec has it. Maybe there is a problem?

**Comment by dario-l** [comment]
Uninstalled and installed again by following docs. Nothing changed. Error is still the same :(

**Comment by aliozgur** [comment]
0.1.1 published on nuget.org. Please try to update or reinstall the package and see if the latest release solves your problem

**Comment by dario-l** [comment]
Unfortunately not. I don't know where the problem may be. :/

**Comment by aliozgur** [comment]
Any update on the error message?

**Comment by dario-l** [comment]
Error message is the same.

**Comment by aliozgur** [comment]
Weird, since all deps were updated with the new version. Any chance for something like Tamviewer or remote access for inspection.

**Comment by aliozgur** [comment]
Project type updated to .NET SDK and Octokit package updated to version 0.50.0
Changes are available version 0.1.2 on NuGet
